apps/nacl_demos/Makefile

```diff
@@ -36,10 +36,10 @@ LDFLAGS_PNACL := -L $(LIBDIR_PNACL) -lppapi_cpp -lppapi
 DEMOS = game_of_life julia reaction_diffusion reaction_diffusion_2
 DEMO_OBJECTS = $(DEMOS:%=%_init) $(DEMOS:%=%_update) $(DEMOS:%=%_render)
 
-OBJECTS_32 = nacl_demos_32.o $(DEMO_OBJECTS:%=build_32/%.o)
-OBJECTS_ARM = nacl_demos_arm.o $(DEMO_OBJECTS:%=build_arm/%.o)
-OBJECTS_64 = nacl_demos_64.o $(DEMO_OBJECTS:%=build_64/%.o)
-OBJECTS_PNACL = nacl_demos_pnacl.bc $(DEMO_OBJECTS:%=build_pnacl/%.bc)
+OBJECTS_32 = nacl_demos_32.o $(DEMO_OBJECTS:%=build_32/%.a)
+OBJECTS_ARM = nacl_demos_arm.o $(DEMO_OBJECTS:%=build_arm/%.a)
+OBJECTS_64 = nacl_demos_64.o $(DEMO_OBJECTS:%=build_64/%.a)
+OBJECTS_PNACL = nacl_demos_pnacl.bc $(DEMO_OBJECTS:%=build_pnacl/%.a)
```

Comment:
Seems odd that the RHS has changed from .bc to .a. Is that correct?

Did you manage to test this change to the nacl demos? Seems like it would be sensitive to this sort of change.